main/main.js

```diff
@@ -54,27 +54,27 @@ app.on('ready', function() {
     var server = express();
 
     // Screenshot を配信
-    server.get("/ss", function(req, res) {
+    server.get('/ss', function(req, res) {
 
-      var options = {
-        dotfiles: 'deny',
-        headers: {
-          'x-timestamp': Date.now(),
-          'x-sent': true
-        }
-      };
+        var options = {
+            dotfiles: 'deny',
+            headers: {
+                'x-timestamp': Date.now(),
+                'x-sent': true
+            }
+        };
 
-      const filename = "/tmp/toggle-board-ss.png";
-      res.sendFile(filename, function(err) {
-        if (err) {
-          console.error(err);
-          res.status(err.status).end();
-        }
-      });
+        const filename = '/tmp/toggle-board-ss.png';
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).